content/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust.md

```diff
@@ -47,7 +79,7 @@ We recommend that you have a basic understanding of the Rust language. For more
 
 ## Specifying a Rust version
 
-At the time of writing, the default rust compiler version is 1.83.0 rustup is available and can be used to install additional toolchains.
+{% data variables.product.prodname_dotcom %}-hosted runners include a recent version of the rustup rust compiler. You can use rustup to report on the version installed on a runner, override the version, and to install additional toolchains. For more information, see [The rustup book](https://rust-lang.github.io/rustup/).
```

Comment:
Revised to remove the reference to an explicit version of rustup to stop the information going out of date.